### DISCUSSION TITLE
How far back can MVT-iOS retrieve data?

### DISCUSSION THREAD

**Discussion by citycritter** [mvt-project/mvt#422] [Q&A] [answered]
I got a new phone last summer and wanted to know if MVT-iOS could go back up to 6 months worth of iPhone data?

**Comment by Te-k** [comment, marked as answer]
Try it and you'll see but generally backups have data since the first installation of the phone and potentially before if a backup was restored from another phone

**Reply by citycritter** [reply]
Ok, I see it had history even before I got the phone (factory setup was logged?) which was a fe months back. I've had the phone for less than a year so that may play a role too.
